### ISSUE TITLE
Add option to disable BetterBags for Bank inventory

### ISSUE THREAD

**Issue by MyDarky** [Cidan/BetterBags#93]
Please add an option to disable BetterBags for Bank inventory, I like the addon but I want my bank to remain as it is. Thank you.

**Comment by Cidan** [comment]
We will add this feature in a future release.

**Comment by jujubejules** [comment]
I'd like to second this. Especially with Warbound tabs, I have to turn BetterBags off if I want to deposit/withdraw currency or modify tabs.

**Comment by kzadams** [comment]
10000% this feature would be really great. Had it on adibags. missing it here. Addon is great :)

**Comment by valcaius** [comment]
Made an account just to upvote this. Only thing stopping me from using this app more; this feature would be amazing!

**Comment by paulbuechner** [comment]
+1

**Comment by Louivatar** [comment]
+1000. Please add the ability to disable the addon for the character bank, bank of reagents and warbound bank to the addon settings. Thank you!

**Comment by Pleasure0102** [comment]
this is desperately needed. Especially since we cannot buy bank tabs now without turning the addon off and then on again

**Comment by Freyrzz** [comment]
What about this feature ? i really need it

**Comment by NadineDefiant** [comment]
I want to also second this. I love the addon very much for my bags but desperately want to not need to use it in my bank.

**Comment by Hinkyz** [comment]
Similar to this, I want the ability to only use the addon for my bank, and leave bags as default Blizzard bags

**Comment by ranmas** [comment]
Yeah, this is really needed.  Not only for being able to buy bank tabs, but also so you can change out what bags are in those tabs.
